### PR TITLE
Update COPYRIGHT

### PR DESCRIPTION
**Pull request by MoonkiHong**
Corresponding to the GitHub issue #2807, I would like to propose the correction of the current COPYRIGHT statement w.r.t. the open source matter.

Signed-off-by: Peter Moonki Hong <moonki1.hong@samsung.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
@lemmaa PTAL

**Comment by lemmaa** [comment]
@MoonkiHong , as you can find at the bottom of https://nnfw.readthedocs.io/en/latest/?badge=latest, We want to include community contributors in copyright.e want to include community contributors in copyright.

Screen Shot 2020-07-02 at 6 20 11 PM

There is no particular preference between 'Samsung Electronics' and 'Samsung Research'. We will follow the guide of the Open Source Group.

So, our candidate for copyright are one of 

Copyright (c) 2020 Samsung Electronics Co., Ltd. & Contributors All Rights Reserved.
Copyright (c) 2020 Samsung Research & Contributors All Rights Reserved.

**Comment by MoonkiHong** [comment]
> @MoonkiHong , as you can find at the bottom of https://nnfw.readthedocs.io/en/latest/?badge=latest, We want to include community contributors in copyright.e want to include community contributors in copyright.
> 
> Screen Shot 2020-07-02 at 6 20 11 PM
> 
> There is no particular preference between 'Samsung Electronics' and 'Samsung Research'. We will follow the guide of the Open Source Group.
> 
> So, our candidate for copyright are one of
> 
> Copyright (c) 2020 Samsung Electronics Co., Ltd. & Contributors All Rights Reserved.
> Copyright (c) 2020 Samsung Research & Contributors All Rights Reserved.

@lemmaa Seems to be "Samsung Electronics Co., Ltd." and that has been already attached into other source codes in Samsung/ONE.

File: COPYRIGHT [hunk 1]
```diff
@@ -1 +1 @@
-Copyright (c) <2019> <Samsung Inner Source Community> All Rights Reserved.
+Copyright (c) 2019 Samsung Electronics Co., Ltd. All Rights Reserved.
```
Comment (lemmaa) [inline COPYRIGHT:1]: @MoonkiHong , reflected your opinion. :)

```suggestion
Copyright (c) 2019 Samsung Electronics Co., Ltd. & Contributors All Rights Reserved.
```

Comment (MoonkiHong) [reply COPYRIGHT:1]: Agree. Thank you for the kind suggestion and revision. :)

Comment (lemmaa) [reply COPYRIGHT:1]: @MoonkiHong , if you agree to this suggestion, you need to push `commit suggestion` to apply. :)

**Pull request merged**